src/System.Reflection/tests/Assembly/Assembly_EmbeddedFiles.cs

```diff
@@ -16,6 +16,10 @@ public class EmbeddedFilesTest
         [Fact]
         public void TestEmbeddedFiles()
         {
+            var resources = typeof(EmbeddedFilesTest).GetTypeInfo().Assembly.GetManifestResourceNames();
+
+            Assert.Contains("EmbeddedImage.png", resources);
+
```

Comment:
In the existing code below, should we be verifying that we can read from the stream and verifying something about what we read?  Or is that handled elsewhere, or unnecessary for some reason?